components/proxy/nginx.conf

```diff
@@ -0,0 +1,32 @@
+user  nginx;
```

Comment:
Does this file relate to [openshift/dockerfiles/nginx.conf](https://github.com/ICTU/quality-time/blob/master/openshift/dockerfiles/nginx.conf)? Can they be combined?

Reply:
probably.